documentation/src/docs/deployment/deployment.md

```diff
@@ -28,9 +28,10 @@ The following prerequisites are required for running Pravega in all modes.
 The following prerequisites are required for running in production. These are only required for running in distributed mode.
 
 - External HDFS 2.7
-- Zookeeper 3.5.4-beta
-- Bookkeeper 4.7.3
+- Zookeeper 3.6.1
+- Bookkeeper 4.9.2
 
+Any version onwards from these versions can be used. These versions are also used by [zookeeper-operator](https://github.com/pravega/zookeeper-operator) and [bookkeeper-operator](https://github.com/pravega/bookkeeper-operator)
```

Comment:
`Any version onwards from these versions can be used`

I would remove this altogether. There is no guarantee that a new version of BK will work without issues with the current code of Pravega. If we cannot guarantee that with the current code, we shouldn't be stating this.

Reply:
Yes Andrei